core/src/main/scala/org/http4s/util/Renderable.scala

```diff
@@ -197,7 +197,8 @@ final case class ChunkWriter(
 ) extends Writer {
 
   override def append(s: String): this.type = {
-    toChunk = (toChunk ++ Chunk.bytes(s.getBytes(charset))).toChunk
+    // This smells wrong
```

Comment:
Is it better for this to be a `SegmentWriter`?

Reply:
Yes, I think so, but I think that would be part of a larger overhaul to use Segment generally.